clouddriver-alicloud/src/main/java/com/netflix/spinnaker/clouddriver/alicloud/model/AliCloudInstance.java

```diff
@@ -26,24 +28,21 @@ public class AliCloudInstance implements Instance {
   private String name;
   private Long launchTime;
   private String zone;
-  private String providerType;
-  private String cloudProvider;
+  final String cloudProvider = AliCloudProvider.ID;
   private HealthState healthState;
   private List<Map<String, Object>> health;
 
+  private Map<String, Object> attributes = new HashMap<String, Object>();
```

Comment:
Don't need the types in the declaration

`private Map<String, Object> attributes = new HashMap<>();`